src/modules/refresh.js

```diff
@@ -1,6 +1,4 @@
-/*eslint-disable*/
 import addElem from './addElement';
-import Task from './classTask';
 
```

Comment:
- [ ] "Clear all completed"  functionality is not working, When I click on the button I'm getting the following error. Please fix this issue and make sure "Clear all completed" button works as expected.

### See the image of the error

![image](https://user-images.githubusercontent.com/83803414/178111521-547f5e46-691d-4f0c-86ca-9f94be4f95c2.png)